### PR TITLE
cmake: format

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by teto** [comment]
how did you format ? with what ? do we have official format rules now ?

**Comment by SuperSandro2000** [comment]
> how did you format ? with what ? do we have official format rules now ?

a little bit of magic, nixpkgs-fmt and myself

**Review by siraben — APPROVED** [review]
This is why I wish we had content addressibility. Nevertheless, LGTM.

**Comment by siraben** [comment]
However should this be combined with another cmake-affecting PR so we don't cause two mass rebuilds?

File: pkgs/development/tools/build-managers/cmake/default.nix [hunk 1]
```diff
@@ -62,27 +61,24 @@ stdenv.mkDerivation rec {
       --subst-var-by libc_bin ${lib.getBin stdenv.cc.libc} \
       --subst-var-by libc_dev ${lib.getDev stdenv.cc.libc} \
       --subst-var-by libc_lib ${lib.getLib stdenv.cc.libc}
-  ''
```
Comment (dasJ) [inline pkgs/development/tools/build-managers/cmake/default.nix:65]: If you didn't do this one change you could get around the mass rebuild btw

Comment (SuperSandro2000) [reply pkgs/development/tools/build-managers/cmake/default.nix:65]: Thats fine for me :)

Comment (siraben) [reply pkgs/development/tools/build-managers/cmake/default.nix:65]: 🙈

**Pull request merged**